src/templates/docker-compose.yml.j2

```diff
@@ -36,6 +36,7 @@ scirius:
     build: $basepath/docker/scirius
     volumes:
         - $basepath/config/scirius:/etc/scirius:ro
+        - $basepath/config/scirius/backups:/var/backups:ro
```

Comment:
It can't be read-only here. Scirius needs to write in the backup.

I think it will be better to put backups in `$basepath/backups` and you will also need to create the directory during the setup.